Dockerfile.client

```diff
@@ -1,6 +1,6 @@
 FROM ocaml/opam:alpine
 
-RUN opam depext lwt && opam install lwt alcotest conf-libev
+RUN opam depext -i lwt alcotest conf-libev
```

Comment:
This should be `-ui` to be reliable in the face of upstream revision updates (it triggers an `apk update` before attempting to install)